src/ctim/schemas/common.cljc

```diff
@@ -255,10 +257,21 @@
                                "general information about security threats might be marked as `amber` or "
                                "`green`, indicating that it can be shared more broadly within an organization.")))))
 
+(def time-metas
+   (f/optional-entries
+    (f/entry :created Time
+             :description "The time this object was created at.")
+    (f/entry :modified Time
+             :description "The time this object was last modified.")))
```

Comment:
`timestamp` can be set in a new entity but not `created`. For a CTIM user, it is not clear what is the difference. Could you clarify the usage in the description? Do we expect to have the same value in `timestamp` and `created` when the entity is new and the same value in `timestamp` and `modified` when it was updated?

Reply:
I updated the doc.
We do set `timestamp` with the value of `created` for incidents when it's not provided.
If it's provided by the source, we keep the provided value, and we update it at each modification in the [default realize](https://github.com/threatgrid/ctia/blob/7bdefcc06f7ec7e74575ed841a92d36ae4a2c1af/src/ctia/domain/entities.clj#L38).
However, since this [PR](https://github.com/threatgrid/ctia/pull/1377) we have a custom realize for incident that do not update timestamp on modification. We [decided](https://github.com/advthreat/iroh/issues/7853#issuecomment-1562811852) that after another confusion, but we did not want to apply it to all entiites (IMO it should be).
So in this context, I hope that my update to the doc still help users of CTIM.
Do you have another proposal?

Reply:
If `created` and `modified` are specific to the application in charge of the persistence, I would only change the schema and expose it in the application.

Reply:
The purpose is to enable it to be consumed by users of the application.
Only the application can modify it, but the information that convey these data can be consumed by API clients and is not specific to the application.
In particular, [we can display it in the UI](https://github.com/advthreat/iroh/issues/8788#issuecomment-1887314105).